configs/hugetlb.go

```diff
@@ -0,0 +1,9 @@
+package configs
+
+type Hugetlb struct {
```

Comment:
We should stick this one in the cgroups file since all the cgroup stuff is there